### PR TITLE
fix(config): add fingerprint to Kwikset 910

### LINKED ISSUE #2775: Missing configuration: Kwikset 910 Z-Wave Plus Lock
**Which device is missing?**

Kwikset 910 Lock, Z-Wave Plus

**What are the IDs?**

Manufacturer ID: 0x0090 
Product type: 0x0003
Product ID: 0x0000 

**Did you check if we really don't have it?**

- [ X] It is not in the [configuration DB](https://devices.zwave-js.io/)
- [ X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)
- [ X] It is not in the [TODO list](https://github.com/zwave-js/node-zwave-js/issues/1600)

Here's an export of the node from the zwavejs2mqtt interface (forgive me if this is useless):
[node_47.json.zip](https://github.com/zwave-js/node-zwave-js/files/6595196/node_47.json.zip)

Basically, I ditched the old z-wave card in favor of a z-wave plus card I purchased on ebay. My coordinator sees the lock and it pairs without a hitch, except it shows up as an unknown mfg and device. The lock responds to unlock and lock commands issued from home assistant. I can also use zwavejs2mqtt to successfully send new user codes. But the alarm_type and status are not being reported at all, so I can't which code was used to unlock the lock.

Any help is greatly appreciated.

s-l1600
s-l1600 (1)
s-l1600 (2)

### PR DESCRIPTION
**Pull request by blhoward2**
Closes #2775

### CODE REVIEW

File: packages/config/config/devices/0x0090/910.json [hunk 1]
```diff
@@ -16,7 +16,11 @@
 		},
 		{
 			"productType": "0x0003",
-			"productId": "0x0238",
+			"productId": "0x0238"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x0000",
 			"zwaveAllianceId": [2003, 2239]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0090/910.json]: Shouldn't this alliance ID be attached to the previous fingerprint?

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0090/910.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0090/910.json]: Yes, fixed.

**Comment by blhoward2** [comment]
@AlCalzone all ready

**Comment by AlCalzone** [comment]
Uhm - am I blind or did you forget to push the commit?

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**